pages/book/cells.mdx

```diff
@@ -113,11 +112,47 @@ contract SerializationExample {
 }
 ```
 
-### `remaining` [#serialization-bytes64]
+### `remaining` [#serialization-remaining]
+
+By default, when it comes to storing values of [`Cell{:tact}`](#cells), [`Builder{:tact}`](#builders) and [`Slice{:tact}`](#slices) types in persistent storage of loading them from the received messages or assigned values, Tact's auto-layout algorithm uses references (refs):
+
+```tact
+contract ReReReF {
+    c: Cell;    // ^cell in TL-B notation
+    b: Builder; // ^builder in TL-B notation
+    s: Slice;   // ^slice in TL-B notation
+}
+```
+
+However, using `remaining{:tact}` makes values assigned to those fields to be stored directly as a [`Slice{:tact}`](#slices):
```

Comment:
I'm having trouble understanding this sentence. And I find all the explanations here overall confusing and lacking clarity. 
And it fails to resolve the issue which asks the following:

> For now, lots of people asking about forwardPayload: Slice as remaining when they dealing with Jetton Token.